backend/api/constants.py

```diff
@@ -1,3 +1,6 @@
 class ApiExecution:
     PATH: str = "deployment/api"
     MAXIMUM_TIMEOUT_IN_SEC: int = 300  # 5 minutes
+    DEFAULT_TIMEOUT_IN_SEC: int = 80
```

Comment:
@muhammad-ali-e @chandrasekharan-zipstack should we keep the default timeout to maximum ?

Reply:
It can be done - however I remember that if the workflow runs within the timeout the API is synchronous and if its long-running we'll treat it as asynchronous and force the user to use the status API.
In this regard will it be a product concern? Anyway I feel that it can be a product call

Reply:
Also, How can we inform the maximum timeout value to user?

Reply:
I think we have validations for the same. So we should be good I think

Reply:
Anyway @ritwik-g do you want me to set the default timeout to the max possible?